examples/management_policies.ipynb

```diff
@@ -0,0 +1,551 @@
+   },
+   "id": "5a1f198c2dbcd50b"
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Setup\n",
+    "\n",
+    "See \"Hello OpenAssetIO\" notebook for details on how to bootstrap OpenAssetIO."
+   ],
+   "metadata": {
+    "collapsed": false
+   },
+   "id": "a5f0c2a3a4a3117d"
+  },
+  {
+   "cell_type": "code",
+   "outputs": [],
+   "source": [
+    "from resources import helpers\n",
+    "\n",
+    "\n",
+    "manager, context = helpers.bootstrap(\"resources/management_policies/openassetio_config.toml\")"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2024-01-30T18:11:38.083252222Z",
+     "start_time": "2024-01-30T18:11:38.027702394Z"
+    }
+   },
+   "id": "320f8ec6b0f6ea91",
+   "execution_count": 28
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Setting the scene"
+   ],
+   "metadata": {
+    "collapsed": false
+   },
+   "id": "b2ab89518162569f"
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Our DCC tool wishes to load a 3D model, render it to a 2D image, then publish that render.\n",
+    "\n",
+    "We wish to tweak the user's workflow to fit with the manager's capabilities, adjusting our UI and entity queries."
+   ],
+   "metadata": {
+    "collapsed": false
+   },
+   "id": "d3a4359d83015d83"
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Determining resolvable geometry properties"
+   ],
+   "metadata": {
+    "collapsed": false
+   },
+   "id": "9a9af3550e53094b"
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "We want to load the geometry, but a common issue with geometry formats is knowing which axis is intended to be used as the \"up\" direction. \n",
+    "\n",
+    "Ideally the manager would tell us which axis to consider as \"up\", but if not we need to show a different UI panel, allowing the user to tweak the base orientation.\n",
+    "\n",
+    "We can decide ahead of time what UI should be shown by querying the manager's policy for this type of entity. This will tell us which traits of the entity that the manager can `resolve`.\n",
+    "\n",
+    "The MediaCreation package has an appropriate Specification for describing 3D geometry, so lets use that."
+   ],
+   "metadata": {
+    "collapsed": false
+   },
+   "id": "e7b86520d4d4dd69"
+  },
+  {
+   "cell_type": "code",
+   "outputs": [
+    {
+     "data": {
+      "text/markdown": "> **Result:**\n> `{'openassetio-mediacreation:threeDimensional.Spatial', 'openassetio-mediacreation:content.LocatableContent', 'openassetio-mediacreation:usage.Entity', 'openassetio-mediacreation:threeDimensional.Geometry'}`"
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from openassetio_mediacreation.specifications.threeDimensional import \\\n",
+    "    SceneGeometryResourceSpecification\n",
+    "\n",
+    "\n",
+    "helpers.display_result(SceneGeometryResourceSpecification.kTraitSet)"
```

Comment:
:thinking: Obvs we can't use `print`, since that's a reserved word.

This change means updating all notebooks, so we should decide on an alternative before I make the change.

It looks like the naming was derived from the `IPython.display.display_markdown` utility function in IPython.

There is also our `display_image`, which I think is OK?

* `format_and_print` is accurate, but a mouthful?
* `display_text`?
* `display_value`? (just trying for consistency with `display_image`)
* ?

Reply:
Yeah of course, i'll make an issue for it which will surely eventually happen for sure.

Reply:
https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation/issues/80